lib/brainstem/presenter_collection.rb

```diff
@@ -226,6 +226,11 @@ def handle_only(scope, only)
 
     def run_filters(scope, options)
       extracted_filters = extract_filters(options)
+
+      params = extracted_filters.each.with_object({}) do |(key, val), hash|
+        hash[key] = val[:arg]
```

Comment:
if they don't have an arg, they're a default filter, right?  can the user send a nil arg?